tests/MIRI_ecfs/S4_MIRI.ecf

```diff
@@ -35,6 +35,7 @@ teff            6000 #effective temperature of the star in K
 logg            4.0 #surface gravity in log g
 exotic_ld_direc /home/User/exotic-ld_data/ #directory for ancillary files for exotic-ld
 exotic_ld_grid  3D #1D or 3D model grid
+exotic_ld_file  /home/User/exotic-ld_data/Custom_throughput_file.csv
```

Comment:
I don't understand why the tests didn't crash since no such file exists, but I'm pretty sure this should be set to None in the testing files